### PR TITLE
[KO]: Fix error tfds.load()

### PR DESCRIPTION
**Pull request by rickiepark**
Change tfds.load() to new TFDS subsplit API
And update several commits since Oct 22, 2019

Apply new TFDS subsplit API:
```
train_data, validation_data, test_data = tfds.load(
    name="imdb_reviews", 
    split=('train[:60%]', 'train[60%:]', 'test'),
    as_supervised=True)
```

Add import:
```
!pip install tensorflow-hub
!pip install tfds-nightly
```

Update datasets link
```
IMDB 데이터셋은 [imdb reviews](https://www.tensorflow.org/datasets/catalog/imdb_reviews) 또는 [텐서플로 데이터셋](https://www.tensorflow.org/datasets)(TensorFlow datasets)에 포함되어 있습니다. ...
```

Remove sigmoid func
```
model.add(tf.keras.layers.Dense(1))
...
model.compile(optimizer='adam',
              loss=tf.keras.losses.BinaryCrossentropy(from_logits=True),
              metrics=['accuracy'])
```

Running in colab is ok.
Thanks!

### CODE REVIEW

**Comment by lamberta** [comment]
Thanks, Haesun. (I saw your picture on the TF Dev Summit livestream 😄)

To make the diffs cleaner, can you try running this through the new [nbfmt.py](https://github.com/tensorflow/docs/blob/master/tools/nbfmt.py) tool? Hope to eventually make it part of the CI, but still testing.

**Comment by rickiepark** [comment]
Thanks @lamberta 
Next time, I want to see your picture! :)

I'll try nbfmt.py soon

**Comment by rickiepark** [comment]
Hi @lamberta ,
It is updated by nbfmt.py and tested again in colab.
Thanks! :)

**Comment by jaketae** [comment]
@rickiepark Thank you for the PR. LGTM!

**Comment by lamberta** [comment]
Fixed merge conflict
